css-positioning/absolute.css

```diff
@@ -46,8 +46,8 @@
   /* your code here */
 
   position: absolute;
-  height: 300px;
-  width: 300px;
+  height: 100%;
+  width: 100%;
   top: 0;
   left: 0;
   z-index: 1;
```

Comment:
The goal here was to use top, right, bottom, and left. The goal is to use top, right, bottom, and left to stretch this across the entire size. The only reason your solution here works with the height is because this element's relative container has a height set to it. If there wasn't a height then this would not work.
```
top: 0;
right: 0;
bottom: 0;
left: 0;
```
This means that the starting point is right at the edge of all 4 sides thus stretching the element across.